javascripts/content_scripts/privly.js

```diff
@@ -763,8 +757,6 @@ var privly = {
 
     "use strict";
 
-    return;
```

Comment:
This is a semantic change. I know CodeClimate will complain about it, but it should not be included in this PR.